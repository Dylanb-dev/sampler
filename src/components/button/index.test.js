/* eslint-disable */

import React from 'react'
import renderer from 'react-test-renderer'

import Button from './'

test('Button renders correctly', () => {
  const props = { text: 'test', onClick: () => {} }

  const tree = renderer.create(<Button {...props} />).toJSON()

  expect(tree).toMatchSnapshot()
})
