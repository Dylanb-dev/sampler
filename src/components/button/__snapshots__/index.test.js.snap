// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
<button
  onClick={[Function]}
  style={
    Object {
      "background": "green",
      "border": "1px solid green",
      "borderRadius": "32px",
      "boxShadow": "0px 8px 15px rgba(0, 0, 0, 0.1)",
      "color": "white",
      "fontSize": "16px",
      "padding": "16px",
      "textShadow": "0 0 0 rgba(255,255,255,0.5)",
      "textTransform": "upperCase",
      "transition": "all 0.3s ease 0s",
    }
  }
>
  test
</button>
`;
